config/dev.toml

```diff
@@ -1,27 +1,33 @@
-[general]
-kafka_topic = 'event-messages' 
-root_path = 'consumed-events'
-flush_interval = 8 # Seconds
-max_output_file_size = 4194304 # 4Mb in bytes
-
 [producer]
```

Comment:
added sub-sections to producer and consumer
dropped general, and split kafka-specific stuff from app-specific